pkg/subctl/cmd/join.go

```diff
@@ -262,10 +262,15 @@ func joinSubmarinerCluster(subctlData *datafile.SubctlData) {
 
 	status.Start("Deploying the Submariner operator")
 
+	// statusReporter are to match submarinerop.Ensure argument requirements.
+	statusReporter := cli.NewReporter()
+	statusReporter.Start("Deploying the Submariner operator")
+
```

Comment:
Why are you creating another CLI reporter in addition to the existing `status`? `cli.Status` implements the new reporter interface.

Reply:
CLI reporter is needed by `submarinerop.Ensure` at L271. Also I am keeping the old `cli.Status` implementation as is and will be adjusted as part of `join` refactor.

Reply:
`cli.Status` implements `reporter.Interface` so `status` can be passed to `submarinerop.Ensure`. `statusReporter` is also a `cli.Status` instance. You don't need `statusReporter`.